readme.md

```diff
@@ -31,6 +31,29 @@ The following keys can be used to control the clock:
 - `c` shows controls (does not work on local file system)
 - `<F11>` toggle fullscreen
 
+Audio soundtracks
+-----
+The soundtracks are saved in the mp3 folder inside the clock. If you want, you can add as many files in there as you want, and you can use them at different times of the clock, or after events in the timeline.
+To chose which files to use, you simply configure the tracks area in the config.js file.
+The following options are avalable:
+`tracks: [{
+	//This track will start when the clock starts.
+	source: 'audio_source_file.mp3',
+	start: 'on start'
+},{
+	//This track will start when the clock ends. If you prefer yuo can use 'one stop' instead.
+	source: 'audio_source_file.mp3',
+	start: 'on start'
+},{
+	//This track will start 30 seconds after the clock starts.
+	source: 'audio_source_file.mp3',
+	start: '30 seconds after start'
+},{
+	//This track will start 30 seconds before the clock ends.
+	source: 'audio_source_file.mp3',
+	start: '30 seconds before stop'
+}]`
+

```

Comment:
If I understood correctly there are more options

- start: 20 seconds
- start: 50%
- start: on end
- start: on pause

The last bit refers to events, right? Then we should tell about which events are available. Is it true that we have basically these formats:

- on [event]
- nn seconds before|after [event]

where [event] can be start|stop|end|pause|toggle|arm|[nn] where [nn] is number of seconds

Reply:
That's almost correct. in the seconds option, you don't have to use seconds,
Those options are equivalent:
200s = 200 secs = 200 seconds
Also
50% = 50 percents
Updated it to reflect that